homeassistant/helpers/event.py

```diff
@@ -161,7 +163,7 @@ def _async_state_change_dispatcher(event: Event) -> None:
             if entity_id not in entity_callbacks:
                 return
 
-            for action in entity_callbacks[entity_id]:
+            for action in entity_callbacks[entity_id][:]:
```

Comment:
This is slightly more readable.

```suggestion
            for action in list(entity_callbacks[entity_id]):
```

Reply:
Ah this is faster. https://stackoverflow.com/a/14821420

Reply:
Either is fine. Let's get it out and see if it helps.

Reply:
Let's go back to the faster version if that's ok